src/backend/routers/auth.py

```diff
@@ -317,3 +317,74 @@ def log_and_redirect_err(error_message: str):
     response = RedirectResponse(redirect_uri)
 
     return response
+
+
+@router.delete("/tool/auth")
+async def delete_tool_auth(
+    request: Request, session: DBSessionDep, ctx: Context = Depends(get_context)
+):
+    """
+    Endpoint to delete Tool Authentication.
+
+    If completed, the corresponding ToolAuth for the requesting user is removed from the DB.
+
+    Args:
+        request (Request): current Request object. Note that the request body should contain the tool_id parameter as the tool to delete.
+        session (DBSessionDep): Database session.
+        ctx (Context): Context object.
+
+    Returns:
+        {"status": "success"} on successful deletion.
+
+    Raises:
+        HTTPException: If there was an error deleting the tool auth.
+    """
+
+    logger = ctx.get_logger()
+
+    def log_and_return_error(error_message: str):
+        logger.error(event=error_message)
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=f"{error_message}",
+        )
+
+    user_id = ctx.get_user_id()
+
+    try:
+        request_body = await request.json()
```

Comment:
hey @chantelle-cohere, a delete shouldn't have a body request. 

> DELETE cannot have request body, just like GET cannot have a request body. (Well, it's a syntactically valid HTTP message, but there's no semantic meaning to the body.)

I think we should pass the Id of the tool as a parameter and keep the delete method.

e.g.:

```
DELETE /tool/auth/<uuid>
```

Reply:
gotcha, thank you! I will update to match your suggestion!